pkg/params/info/events.go

```diff
@@ -29,6 +29,8 @@ type Event struct {
 	SHATitle          string // commit title for UIs
 	PullRequestNumber int    // Pull or Merge Request number
 
+	TestPipelineRun string // PipelineRun name to run
```

Comment:
maybe we should start break things apart in info.Event as it gets crowded, what about : 

```go
Events {
 State {
   TargetTestPipelineRun string
  }
}
```

(we should probably refactor the full struct one day)

Reply:
yeah we can move provider specific fields in their own struct
and refer in Event

Reply:
and may be have provider name prefix so would be easy to know

Reply:
we can refactor the provider struct to their own, ie like https://github.com/openshift-pipelines/pipelines-as-code/blob/main/pkg/provider/gitlab/gitlab.go#L40-L44